README.md

```diff
@@ -324,6 +324,31 @@ print(json.dumps(report, indent=4))
 
 Note that if merged profiles had overlapping integer indices, when null rows are calculated the indices will be "shifted" to uninhabited values so that null counts and ratios are still accurate.
 
+### Profiler Differences
+For finding the change between profiles with the same schema we can utilize the
+profile's `diff` function. The diff will provide overall file and sampling 
+differences as well as detailed differences of the data's statistics. For 
+example, numerical columns have a t-test applied to evaluate similarity.
+More information is described in the [Profiler Github Pages](
+https://capitalone.github.io/DataProfiler/docs/0.7.1/html/profiler.html#profile-differences).
```

Comment:
Just link to docs and describe... it changes versions. Unfortunately there's no way to link.

Reply:
done

Reply:
good call